### PR TITLE
Set Spotlight of the Week in Panel

### PR DESCRIPTION
**Pull request by anmol678**
Added a dropdown to Manger tab in Panel to select the Spotlight Show of the Week

screen shot 2017-05-10 at 11 30 55 pm
screen shot 2017-05-10 at 11 31 04 pm

To do:
1. Modify API to return nowPlaying and spotlight along with shows on route `/api/schedule`
2. Modify database to store spotlightShow
3. Render Spotlight to Frontpage
Refer to #116 for more.

### CODE REVIEW

**Comment by anmol678** [comment]
#### Update
InputSpotlightDropdown supports search in dropdown.

screen shot 2017-05-30 at 3 32 45 pm
screen shot 2017-05-30 at 3 33 12 pm

File: react/panel/inputs/InputSpotlightDropdown.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// InputSpotlightDropdown.jsx
+import 'react-select/dist/react-select.css'; //not sure where to put this!
```
Comment (binerys) [inline react/panel/inputs/InputSpotlightDropdown.jsx]: Let's keep this here for now!

File: react/panelManager.jsx [hunk 2]
```diff
@@ -57,6 +59,66 @@ var ManagerPage = React.createClass({
...
+
+var Spotlight = React.createClass({
+  handleSpotlightSubmit: function(showId) {
+    console.log(showId);
+    // Push to server (this.props.urls.spotlightShow)
+  },
+  render: function () {
+    return (
+      <div>
+        <h4>Spotlight show of the Week</h4>
```
Comment (binerys) [inline react/panelManager.jsx]: Can we change this to: Spotlight Show of the Week

File: react/panel/inputs/InputSpotlightDropdown.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        label: show.title
+      }];
+    });
+    var clearable = false;
+
+    return <Select options={options} value={this.state.spotlightShow} onChange={this.handleSpotlightChange} clearable={clearable} />;
+  }
+});
+
+module.exports = Dropdown;
```
Comment (binerys) [inline react/panel/inputs/InputSpotlightDropdown.jsx]: Instead of calling it just `Dropdown`, let's call it: `InputSpotlightDropdown` so its more specific

**Comment by anmol678** [comment]
No, I won't be adding a submit button. It sets to whatever is selected.

**Comment by binerys** [comment]
Closing due to work being moved to: https://github.com/uclaradio/uclaradio-api

**Pull request closed without merging**